test/acceptance/helpers/index.js

```diff
@@ -138,6 +139,15 @@ const startGravity = (port) =>
         expires_in: moment().add(100, 'days').utc().format()
       })
     })
+    app.get('/api/v1/profile/pace-gallery', (req, res) => {
+      res.send(fabricate('profile', {
+        owner_type: 'PartnerGallery',
+        owner: fabricate('partner', {
+          id: 'pace-gallery',
+          profile_layout: 'gallery_one'
+        })
+      }))
+    })
```

Comment:
I added this route to test the profile page properly showing an active partner's contact info. This seems too specific to live in this file while others (access token, `/me`) are more generic, and I'm wondering if there's a better place for it to live other than this setup file.